x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings/latest_findings_table.test.tsx

```diff
@@ -19,21 +19,40 @@ const chance = new Chance();
 type TableProps = PropsOf<typeof FindingsTable>;
 
 describe('<FindingsTable />', () => {
-  it('renders the zero state when status success and data has a length of zero ', async () => {
+  const renderWrapper = (opts?: Partial<TableProps>) => {
     const props: TableProps = {
       loading: false,
-      items: [],
+      items: opts?.items || [],
       sorting: { sort: { field: '@timestamp', direction: 'desc' } },
-      pagination: { pageSize: 10, pageIndex: 1, totalItemCount: 0 },
+      pagination: { pageSize: 10, pageIndex: 1, totalItemCount: opts?.items?.length || 0 },
       setTableOptions: jest.fn(),
       onAddFilter: jest.fn(),
+      ...opts,
     };
 
     render(
       <TestProvider>
         <FindingsTable {...props} />
       </TestProvider>
     );
+    return props;
+  };
+
+  it('opens/closes the flyout when clicked on expand/close buttons ', () => {
+    renderWrapper({ items: [getFindingsFixture()] });
+
+    expect(screen.queryByTestId('findings_flyout')).not.toBeInTheDocument();
```

Comment:
[turns out](https://stackoverflow.com/questions/52783144/how-do-you-test-for-the-non-existence-of-an-element-using-jest-and-react-testing) we need to use `queryByTestId` and not `getByTestId` for `.not` stuff because it throws.